docs/examples/data_connectors/simple_directory_reader.ipynb

```diff
@@ -52,20 +52,69 @@
    ]
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    for docs in reader.iter_data():
</pre><p>if a folder has nested folder, we better to use iter_data as default?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9658/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
it's not related to nested data actually

The SimpleDirectoryReader gets a giant list of files from a directory. You can either wait for it to load all data, or use `iter_data()` and process files as they load, to help with more interactive workloads